string position_name
---
float64 x
float64 y
string cardinal_point